fn-extra/src/Web/Fn/Extra/Heist.hs

```diff
@@ -89,11 +93,16 @@ heistInit :: HeistContext ctxt =>
              IO (Either [String] (FnHeistState ctxt))
 heistInit templateLocations isplices csplices =
   do let ts = map (loadTemplates . T.unpack) templateLocations
-     runEitherT $ initHeist (emptyHeistConfig & hcTemplateLocations .~ ts
-                                    & hcInterpretedSplices .~ isplices
-                                    & hcLoadTimeSplices .~ defaultLoadTimeSplices
-                                    & hcCompiledSplices .~ csplices
-                                    & hcNamespace .~ "")
+     let config = emptyHeistConfig & hcTemplateLocations .~ ts
+                                   & hcInterpretedSplices .~ isplices
+                                   & hcLoadTimeSplices .~ defaultLoadTimeSplices
+                                   & hcCompiledSplices .~ csplices
+                                   & hcNamespace .~ ""
+     #if MIN_VERSION_heist(1,0,0)
+     initHeist config
+     #else
```

Comment:
I'm getting a `parse error on input 'else'`. It seems to go away if the CPP directives aren't indented. I'm building this in a Cabal sandbox (not using `stack`) with GHC 8.0.2.